src/plugins/banner/public/plugin.ts

```diff
@@ -21,13 +22,16 @@ export class BannerPlugin implements Plugin<BannerPluginSetup, BannerPluginStart
   }
 
   public async start(core: CoreStart): Promise<BannerPluginStart> {
-    // Render the banner component and pass the HTTP client
-    renderBanner(core.http);
+    // Set the global banner in the chrome service
+    core.chrome.setGlobalBanner({
+      component: React.createElement(GlobalBanner, { http: core.http }),
+    });
 
     return {};
   }
 
   public stop() {
-    unmountBanner();
+    // The banner will be automatically removed when the plugin is stopped
+    // because the component will be unmounted
```

Comment:
do we need the comment?

Reply:
What causes the component to be unmounted on stop?